src/masonite/validation/Validator.py

```diff
@@ -1029,6 +1029,46 @@ def negated_message(self, attribute):
         return "The {} is a valid {} postal code.".format(attribute, self.locale)
 
 
+def parse_table(table_name, column="id"):
+    import importlib
+    if "." in table_name:
+        m = importlib.import_module(table_name)
+        # get model
+        model_name = table_name.split(".")[-1]
+        model = getattr(m, model_name)
+        table = model().get_table()
+        return table, model
+    else:
+        return table_name, None
+
+
+class exists_in_db(BaseValidation):
+    def __init__(self, validations, table, column=None, messages={}, raises={}):
+        super().__init__(validations, messages=messages, raises=raises)
+        # retrieving db connection will totally change with new ORM
+        # in addition, below is not really satisfying
+        from masonite.helpers import config
+        self.connection = config("database.db")
```

Comment:
@josephmancuso 
(even if it has not yet be upgraded to ORM)
I was doing this: fetch the connection from the config and then prepare a query based on the validator parameters

Reply:
get its not upgraded. To make this work you would have to do:

```
self.connection = config("database.db").get_query_builder()
```

Then you can make queries

```python
self.connection = config("database.db").get_query_builder()
self.connection.table(table).where(..)
```